quarkus-extension/engine/runtime/src/main/java/org/camunda/bpm/quarkus/engine/extension/CamundaEngineConfig.java

```diff
@@ -16,36 +16,33 @@
  */
 package org.camunda.bpm.quarkus.engine.extension;
 
-import java.util.Map;
-import java.util.Optional;
-
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import java.util.Map;
+import java.util.Optional;
 
-@ConfigRoot(phase = ConfigPhase.RUN_TIME, name = "camunda")
-public class CamundaEngineConfig {
+@ConfigMapping(prefix = "quarkus.camunda")
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public interface CamundaEngineConfig {
```

Comment:
Migrated to interface following the guidelines of the [Migration Guide 3.14](https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.14#:~:text=While%20we%20encourage%20you%20to%20move%20to%20the%20new%20%40ConfigMapping%20interface%20approach)

Reply:
🎉 I like looking at this! Interface FTW 😄